label/src/Label.js

```diff
@@ -1,14 +1,16 @@
 import copyTextToClipboard from 'copy-to-clipboard';
-import captionStyles from './Caption.css';
+import labelStyles from './label.css';
```

Comment:
we have a convention in this project to call component file with a capital letter as well as it's styles file